app/services/restream.ts

```diff
@@ -303,6 +309,17 @@ export class RestreamService extends StatefulService<IRestreamState> {
     this.chatView.destroy();
     this.chatView = null;
   }
+
+  private handleSettingsChanged(changed: Partial<ICustomizationServiceState>) {
+    if (!this.chatView) return;
+    if (changed.chatZoomFactor) {
+      this.chatView.webContents.setZoomFactor(changed.chatZoomFactor);
+    }
+
+    if (changed.enableBTTVEmotes != null || changed.enableFFZEmotes != null) {
```

Comment:
I think this part shouldn't be needed for multistream chat.

Reply:
removed